.github/workflows/ci.yml

```
@@ -15,6 +15,8 @@ jobs:
         uses: actions/checkout@v2
       - name: Bundle Install
         run: bundle install
+      - name: Select Xcode Version
```

nit (here and below):
```suggestion
      - name: Select Minimum Supported Xcode Version
```

---

Heh, right now this isn't actually using the minimum supported version. I'll take another pass at the CI config once we figure out what the minimum support Xcode version will end up being.